### ISSUE TITLE
Loader issue

### ISSUE THREAD

**Issue by DingLiSim** [chrisrzhou/react-wordcloud#39]
I'm fairly new to webpack and seem to have run into trouble when I've updated the package to 1.2.0. I used` npm install react-wordcloud `and had no troubles before with version 1.1.1. On my webpack config, js and jsx files are using only `babel-loaders`. Am I missing a configuration on the project? 

Babel settings
```
"babel-core": "^6.26.0",
"babel-eslint": "^8.1.2",
"babel-loader": "^7.1.1",
```

```./node_modules/react-wordcloud/dist/index.module.js
Module parse failed: Unexpected token (758:28)
You may need an appropriate loader to handle this file type.
|   words
| }) => {
|   const mergedCallbacks = { ...defaultCallbacks,
|     ...callbacks
   };
```

**Comment by DingLiSim** [comment]
I'm not sure if it had something to do with the babel spread syntax? I've checked the babel updated version for the project and it seems like a highly likely cause?
https://babeljs.io/docs/en/v7-migration
https://babeljs.io/docs/en/babel-plugin-proposal-object-rest-spread

**Comment by chrisrzhou** [comment]
I believe I might have screwed up something with the build that doesn't account for all scenarios.  Let me reproduce this over the weekend.  

It would be tremendously helpful if you can have a Github repo or Codesandbox reproducing this problem that I can clone and test.

Thank you and sorry for the inconvenience.  Please use `v1.1.1` for now to be unblocked.

**Comment by DingLiSim** [comment]
So sorry for the wait @chrisrzhou !!!! I couldn't give you the repo because it was a private repo for clients. I will try my best to make an example repo when I can to reproduce the issue!

**Comment by DingLiSim** [comment]
Hi @chrisrzhou , I've created a sample sandbox with the same config but there seems to be another issue with React.useRef. Perhaps it's because of react-app babel preset settings. My project is using about the same configurations. Do let me know if there's anything I can help out with. 

https://codesandbox.io/s/react-mobx-sandbox-z5u5z

**Comment by chrisrzhou** [comment]
Hey @DingLiSim, not sure what the issue is you are facing.  In the Codesandbox link you provided, seems like the wordcloud is rendering fine!  Let me know if there's some issue you are facing and have a Codesandbox that reproduces the error.  Thanks.  Closing the ticket for now!
